web/vasilemaftita/html/vreme/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>VREME</title>
```

Comment:
try to use English when you exercise HTML tags.

Reply:
Yes, I speak English, a little. tanks.